packages/workflow-engine-redis/src/services/workflow-orchestrator.ts

```diff
@@ -129,6 +145,11 @@ export class WorkflowOrchestratorService {
       container,
     } = options ?? {}
 
+    if (this.isShuttingDown) {
+      // don't accpet any more events
+      return
+    }
```

Comment:
question: Will the workflow run get retried if we hit this case?

Would be great if we could somehow test this flow. It's pretty crucial